### PR TITLE
Mailchimp Campaign Defaults UI component.

### PR DESCRIPTION
**Pull request by budzanowski**
### Information

This is a part of #17434 PR that represents MailChimp Campaign Defaults UI view. This is the step number of the setup procedure.
This is provided for review as a separate PR because the original (spike) PR is too big for the review to be comfortable. 

To test this component in action please refer to #17434.

image

### CODE REVIEW

**Comment by justinshreve** [comment]
I'm not a fan of asking for language here again, especially after the locale setting, and definitely not as a free form input. Can we use the site language for this or at least base it off the choice in the previous step?

File: client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/campaign-defaults.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		{ name: 'campaign_from_email', label: translate( 'From Email' ) },
+		{ name: 'campaign_subject', label: translate( 'Subject' ) },
+		{ name: 'campaign_language', label: translate( 'Language' ) },
+		{ name: 'campaign_permission_reminder', label: translate( 'Permission reminder' ) },
+	];
+
+	return (
+		<div>
+			<div className="setup-steps__campaign-defaults-title">
+				{ translate( 'Campaign default values' ) }
```
Comment (justinshreve) [inline client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/campaign-defaults.js]: Maybe `Campaign Email Settings` instead? To make this step a bit clearer.

**Comment by budzanowski** [comment]
I have removed language setting in this view. The value from the previous step is used silently.

**Comment by budzanowski** [comment]
Corrections are don. Good to go?

**Pull request merged**